docker/Dockerfile.cuda111.x86_64.deps

```diff
@@ -9,7 +9,8 @@ RUN curl -LO https://developer.download.nvidia.com/compute/cuda/11.1.1/local_ins
     ./cuda_*.run --silent --no-opengl-libs --toolkit && \
     rm -f cuda_*.run;
 
-RUN apt-get update && \
+RUN CUDA_VERSION=0.1.0 && \
```

Comment:
Why do you need this? Looks weird.

Reply:
Because docker caches its content based on the Dockerfile commands. So if the installation command is the same even if the underlying package is updated in the server docker build command will use the cached layer. Now, whenever we want to update nvJPEG2000 installation we will change this value and ask the docker to reinstall the necessary package.

Reply:
Can we name it NVJPEG2K_VERSION? I presume, it's that.

Reply:
My bad. Fixed.